Design Tokens Manager - Adds "content sizing" to scale generator results

halocline opened the pull request:
<!--- Provide a general summary of the PR in the Title above -->


<!--- Insert the PR's # for the deploy preview's URL -->
[Deploy Preview](https://deploy-preview-INSERT_PR_#_HERE--keen-mayer-a86c8b.netlify.app/)

#### What does this PR do?

#### Where should the reviewer start?

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

#### How should this be manually tested?

#### Any background context you want to provide?

#### What are the relevant issues?

#### Screenshots (if appropriate)

#### Should this PR be mentioned in Design System updates?

#### Is this change backwards compatible or is it a breaking change?

taysea commented on design-tokens-manager/src/App.jsx:
```diff
@@ -49,7 +54,7 @@ function App() {
               icon={darkMode ? <Moon /> : <Sun />}
               onClick={() => setDarkMode(!darkMode)}
-              tip={`Swith to ${darkMode ? 'light' : 'dark'} mode`}
+              tip={`Switch to ${darkMode ? 'light' : 'dark'} mode`}
```
Thank you.

taysea commented:
<img width="1713" alt="Screenshot 2024-12-06 at 4 28 21 PM" src="https://github.com/user-attachments/assets/6f6016b4-4344-4262-a72b-0950894fd08b">

It feels a little awkward how the "hide scale settings" and copy button wrap below

taysea commented:
> <img alt="Screenshot 2024-12-06 at 4 28 21 PM" width="1713" src="https://private-user-images.githubusercontent.com/12522275/393451516-6f6016b4-4344-4262-a72b-0950894fd08b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM1MzE3MjIsIm5iZiI6MTczMzUzMTQyMiwicGF0aCI6Ii8xMjUyMjI3NS8zOTM0NTE1MTYtNmY2MDE2YjQtNDM0NC00MjYyLWE3MmItMDk1MDg5NGZkMDhiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjA3VDAwMzAyMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMyYjUyMzEwYzA4ZTgzZGUyMGE0Y2E0ZjZiYzdhODQ4NjUzMWUyZTE2NDc3OWU0MmUzMTI5NmY5YzA4YTliNmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.cwtajTcoPkRPcq9OTg1m_kQJCgjqWEySYKJM-AAFK4k">
> It feels a little awkward how the "hide scale settings" and copy button wrap below

Also, I'm assuming the content scale stopped at "xlarge" because I only could chose "20" steps for my primitive scale and there was nothing bigger to get to?

halocline commented:
> It feels a little awkward how the "hide scale settings" and copy button wrap below

Fixed.

halocline commented:
> Also, I'm assuming the content scale stopped at "xlarge" because I only could chose "20" steps for my primitive scale and there was nothing bigger to get to?

Yes, that is what was happening. --> I made a couple enhancements to:
1. Changed steps to be a RangeInput
2. Added more options to the number steps
3. Clear out the content sizes when a content base value is undefined

taysea reviewed (approved):
Refinements look good, I like the rangeinput.
